app/components/CrashDiagramCard.tsx

```diff
@@ -2,63 +2,165 @@ import { useState } from "react";
 import Alert from "react-bootstrap/Alert";
 import Card from "react-bootstrap/Card";
 import Image from "react-bootstrap/Image";
+import Button from "react-bootstrap/Button";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import { BsZoomIn, BsZoomOut } from "react-icons/bs";
+import { SlActionUndo } from "react-icons/sl";
+import {
+  TransformWrapper,
+  TransformComponent,
+  useControls,
+} from "react-zoom-pan-pinch";
 import { Crash } from "@/types/crashes";
 
-// import axios from "axios";
-// import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
-
 const CR3_DIAGRAM_BASE_URL = process.env.NEXT_PUBLIC_CR3_DIAGRAM_BASE_URL!;
 
```

Comment:
So, here's a style question, I have no idea. Is it good style to include these "sub-components", if you will, in this file? They are only going to be used here, so it seems to me it's good to keep them here where in the context of using them. Why pull them out if they will never be used anywhere else?

Obviously -- I will refactor this in a heartbeat based on y'all's advice.

Reply:
personally, i don't mind having them in here. if they grew in size or if other's prefer not, we can stash things in `./utils` and `./configs`. 

i know there has been a lot of though put into file org in Moped recently too—i am not married to this convention if we want to switch things up 🙏

Reply:
OK cool - let's leave them in here so this functionality remains contained in this file and no one has to hunt for anything. If we change our mind at any time, no big, and we can shuffle it around then.